docs/apb_cli.md

```diff
@@ -37,30 +37,37 @@ care of the details so they should be easy to deploy.
 
 [Docker](https://www.docker.com/) must be correctly installed and running on the system.
 
+NOTE: If you are working with a minishift cluster and the [ansible-service-broker addon](https://github.com/minishift/minishift-addons/tree/master/add-ons/ansible-service-broker),
+you must first configure your shell to use the minishift Docker daemon. This can
+be done with `eval $(minishift docker-env)`. For more info, see [minishift's documentation for working with its docker registry](https://docs.openshift.org/latest/minishift/using/docker-daemon.html).
+
 #### Running from a container
 
 Pull the container:
 ```bash
 docker pull docker.io/ansibleplaybookbundle/apb-tools
 ```
+There are three tags to choose from:
+- **latest**: more stable, less frequent releases
+- **nightly**: following upstream commits, installed from RPM
+- **canary**: following upstream commits, installed from source build
 
-Create an alias in your `.bashrc` or somewhere else sane for your shell:
-```bash
-alias apb='docker run --rm --privileged -v $PWD:/mnt -v $HOME/.kube:/.kube -v /var/run/docker.sock:/var/run/docker.sock -u $UID docker.io/ansibleplaybookbundle/apb-tools'
+Copy the [apb-docker-run.sh](../scripts/apb-docker-run.sh) script into your `PATH` and
+make sure it's executable:
+
+```
+cp $APB_CHECKOUT/scripts/apb-docker-run.sh $YOUR_PATH_DIR/apb && chmod +x $YOUR_PATH_DIR/apb
 ```
 
 You should be able to start working by running `apb init my_apb`. The first run may take awhile if you did not pull the image.
 
+> TODO: Will the following actually work?
```

Comment:
I was originally going to suggest removing this, but I think it makes sense for this to stay.

Reply:
atomic run is essentially the same thing as docker run, so it should work.

Reply:
We can remove the ```atomic run``` command. Folks that use atomic will be aware how to run this on atomic since it's s/atomic/docker/.